src/System.Resources.Reader/src/System/Resources/ResourceReader.cs

```diff
@@ -0,0 +1,487 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Resources
+{
+    using System;
+    using System.IO;
+    using System.Globalization;
+    using System.Collections;
+    using System.Text;
+    using System.Threading;
+    using System.Collections.Generic;
+    using System.Diagnostics;
+    public sealed class ResourceReader : IDisposable
```

Comment:
Yeah, it's just copied. I will be doing additional cleanup later. I wanted this PR to include just the split, so there is no confusion as to what code changes are actually being made.